src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java

```diff
@@ -0,0 +1,118 @@
+package com.cloudblue.connect.api.models.product;
+
+import com.cloudblue.connect.api.models.CBCRequestParamConstraints;
+import com.cloudblue.connect.api.models.common.CBCEntity;
+
+import com.cloudblue.connect.api.models.common.CBCEvents;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CBCProductParameter implements CBCEntity {
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String title;
+
+    @JsonProperty
+    private String description;
+
+    @JsonProperty
+    private String type;
+
+    @JsonProperty
+    private String scope;
+
+    @JsonProperty
+    private String phase;
```

Comment:
Should be of type enum with possible values.

Reply:
Done